moped-editor/src/queries/project.js

```diff
@@ -401,6 +407,27 @@ export const ADD_PROJECT_MILESTONE = gql`
   }
 `;
 
+export const PROJECT_FOLLOW = gql`
+  mutation FollowProject(
+    $objects: [moped_user_followed_projects_insert_input!]!
+  ) {
+    insert_moped_user_followed_projects(objects: $objects) {
```

Comment:
I'm not sure if it really matters and I don't know if its worth refactoring but since we are only ever inserting one row of data, I think the `insert_moped_user_followed_projects_one` mutation would be most appropriate. It only takes one object as its argument instead of an array of objects.

![Screen Shot 2022-04-28 at 6 45 02 PM](https://user-images.githubusercontent.com/5697474/165864967-0e7bac50-927a-498c-a9bc-d745297e2a04.png)

Reply:
Done!